src/main/drivers/flash/flash_w25m.c

```diff
@@ -36,8 +36,8 @@
 
 #include "common/maths.h"
 #include "drivers/bus_spi.h"
-#include "drivers/flash.h"
-#include "drivers/flash_impl.h"
+#include "flash.h"
+#include "flash_impl.h"
```

Comment:
drivers/flash

Reply:
They are in the same directory, but for consistency.